src/main/java/com/example/demo/controller/DemoController.java

```diff
@@ -11,4 +13,35 @@ public String index() {
 		return "Hello World ! Greetings from Spring Boot!";
 	}
 
+	@GetMapping(value = "/json", produces = MediaType.APPLICATION_JSON_VALUE)
+	public Hello json() {
+		return new Hello("123", "Raj");
+	}
+
+	private class Hello {
```

Comment:
Controller class should only have the API mapping . Model classes should be separately placed in the model package .